### PR TITLE
Merge 'SLE-15-SP2' into 'SLE-15-SP3'

### PR DESCRIPTION
**Pull request by teclator**
It merges #1291 & #1293 into SLE-15-SP3

### CODE REVIEW

File: src/lib/y2storage/filesystems/blk_filesystem.rb [hunk 1]
```diff
@@ -406,6 +401,33 @@ def preferred_mount_by
...
+
+      # Convenience method for checking if the given STRING is a well formed label
+      #
+      # @param spec [String] content of the first column of an /etc/fstab entry
+      def label?(spec)
+        /^LABEL="(.*)"/ =~ spec || /^LABEL='(.*)'/ =~ spec || /^LABEL=(.*)/ =~ spec
+      end
+
+      # @return [Boolean]
+      def match_uuid?
```
Comment (ancorgs) [inline src/lib/y2storage/filesystems/blk_filesystem.rb:423]: I don't like the idea of using `#last_match` here if the matching actually happened in another method. This forces you to call `#match_uuid?` always right after calling `#uuid?` and only if that call succeeded. Too fragile.

Comment (teclator) [reply src/lib/y2storage/filesystems/blk_filesystem.rb:423]: Yep, have the same opinion

Comment (ancorgs) [reply src/lib/y2storage/filesystems/blk_filesystem.rb:423]: I would have expected something like this:

```ruby
def match_fstab_spec?(spec)
  return true if match_fstab_uuid?(spec)
  return true if match_fstab_label?(spec)

  named_device = devicegraph.find_by_any_name(spec)
  blk_devices.include?(named_device)
end
```

Comment (teclator) [reply src/lib/y2storage/filesystems/blk_filesystem.rb]: After modifying the Regexp we do not need any of these changes at all.

**Review by dgdavid — APPROVED** [review]

**Pull request merged**